gp2gp-translator/src/test/java/uk/nhs/adaptors/pss/translator/generator/BundleGeneratorTest.java

```diff
@@ -0,0 +1,43 @@
+package uk.nhs.adaptors.pss.translator.generator;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.when;
+
+import org.hl7.fhir.dstu3.model.Bundle;
+import org.hl7.fhir.dstu3.model.ResourceType;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import uk.nhs.adaptors.pss.translator.service.FhirIdGeneratorService;
+
+@ExtendWith(MockitoExtension.class)
+public class BundleGeneratorTest {
+
+    @Mock
+    private FhirIdGeneratorService fhirIdGeneratorService;
+
+    @InjectMocks
+    private BundleGenerator bundleGenerator;
+
+    private static final String META_PROFILE = "https://fhir.nhs.uk/STU3/StructureDefinition/GPConnect-StructuredRecord-Bundle-1";
+    private static final String BUNDLE_ID = "219b7cb8-da9e-447a-b279-82be7a3299da";
+
+    @BeforeEach
+    public void setUp() {
+        when(fhirIdGeneratorService.generateUuid()).thenReturn(BUNDLE_ID);
+    }
+
+    @Test
+    public void When_GeneratingBundleResource_Expect_BundleResourceJson() {
```

Comment:
test method name is not accurate
can you please look at other tests in the project and use a similar approach?